2025-09-05 15:32:22,289 - large_scale_cognitive_test_client_8 - INFO - Starting client 8 with config: {'client_id': 8, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 464, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:32:26,455 - large_scale_cognitive_test_client_8 - INFO - Client 8 initialized with 6078 training samples
2025-09-05 15:32:26,458 - large_scale_cognitive_test_client_8 - INFO - Client 8 connecting to 0.0.0.0:8080
2025-09-05 15:32:26,459 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:32:26,489 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:32:26,490 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:32:26,491 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:32:26,697 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:32:26,698 - large_scale_cognitive_test_client_8 - ERROR - Client 8 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:38:39,467 - large_scale_cognitive_test_client_8 - INFO - Starting client 8 with config: {'client_id': 8, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 464, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:38:44,008 - large_scale_cognitive_test_client_8 - INFO - Client 8 initialized with 6078 training samples
2025-09-05 15:38:44,010 - large_scale_cognitive_test_client_8 - INFO - Client 8 connecting to 0.0.0.0:8080
2025-09-05 15:38:44,011 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:38:44,043 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:38:44,044 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:38:44,047 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:38:44,250 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:38:44,251 - large_scale_cognitive_test_client_8 - ERROR - Client 8 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:41:25,022 - large_scale_cognitive_test_client_8 - INFO - Starting client 8 with config: {'client_id': 8, 'server_address': '20.169.237.127:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 464, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:41:29,166 - large_scale_cognitive_test_client_8 - INFO - Client 8 initialized with 6078 training samples
2025-09-05 15:41:29,167 - large_scale_cognitive_test_client_8 - INFO - Client 8 connecting to 20.169.237.127:8080
2025-09-05 15:41:29,168 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:41:29,207 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:41:29,208 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:41:29,209 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:41:29,613 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:41:29,821 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:41:29,821 - large_scale_cognitive_test_client_8 - ERROR - Client 8 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
